docker-compose.yml

```diff
@@ -1,25 +1,10 @@
-version: '2'
```

Comment:
This file needs more work - there is for example no config file being loaded. The environment variables also seem to be outdated

Reply:
I somehow missed adding these changes to the commit, will add and push! :D

Reply:
Alright, will check and add it!